salesforcecli/sfdx-cli
# Sm/promote

### What does this PR do?
promote scripts

### What issues does this PR fix or reference?
@W-11584744@, @W-11587396@, @W-11586924@

### qa
manual promote (for rollback): https://github.com/salesforcecli/sfdx-cli/actions/runs/3008174718 tests most of the pieces

left to qa during release
- slack notifications (goes to alerts channel temporarily, fix that after we see it working correctly)
- the rc-specific scheduled promote

other changes were made to consolidate `version-info` logic in the workflows repo.  This *could* impact the rc-build, particularly dockerSlim

you could verify promote with `sfdx update ci-test` and `npm view sfdx-cli@ci-test` and using the [docker](https://hub.docker.com/r/salesforce/salesforcedx/tags) tags for ci-test
